lib/connection/result/result.js

```diff
@@ -701,6 +701,11 @@ Result.prototype.getStatementId = function ()
   return this._statementId;
 };
 
+Result.prototype.getQueryId = function ()
```

Comment:
Just asking - we need to have this value on:
1. prepared statements
2. SELECT queries
3. Non-SELECT (DML, DDL etc) queries
I see only two new functions. Can you confirm, that it is available for all cases?

Reply:
Hello, it works in all three conditions.